packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -516,6 +516,54 @@ describe('HTS Precompile Acceptance Tests', async function () {
     });
   });
 
+  // FIXME
```

Comment:
Move the PR to draft following steps to address comments, then we can pick it up next sprint.